editor/src/uuiui/widgets/popup-list/popup-list.tsx

```diff
@@ -544,13 +556,24 @@ export const PopupList = betterReactMemo<PopupListProps>(
           container,
           indicatorsContainer: (base) => ({
             ...base,
-            width: 11,
-            height: OptionHeight,
+            width: 14,
+            flexBasis: 14,
+            background: UtopiaTheme.color.inputBackground.shade(102).value,
+            // the control is on top of the input not inside it, so need to make space for input borders
+            height: OptionHeight - 2,
+            marginRight: 1,
+            marginTop: 1,
             padding: 0,
-            marginRight: 6,
+            display: 'flex',
             alignItems: 'center',
             justifyContent: 'center',
             flexShrink: 0,
+            '&:hover': {
+              filter: 'brightness(.98)',
+            },
+            '&:active': {
+              filter: 'brightness(.95)',
+            },
```

Comment:
Have you checked that the changes in this file work with all other cases of the popup-list in the editor?

Reply:
@Rheeseyb Yep, the changes are actually bug fixes.